MAUI/DataForm/floating-label-layout.md

```diff
@@ -0,0 +1,747 @@
+---
+layout: post
+title: Floating label layout in .NET MAUI DataForm control | Syncfusion
+description: Learn about the Floating label layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Floating label Layout in .NET MAUI DataForm (SfDataForm)
+
+## Overview
+
+The floating label layout is a powerful feature that enhances the user experience and makes it easier for users to interact with forms.The dataform supports floating label layout which includes assistive labels, leading and trailing icons, and a password toggle icon to show or hide a password. It offers three different kinds of containers, including filled, outlined, and none. Setting "LayoutType" to "TextInputLayout" enables floating label layout.
+
+## Layout type
+By default, the dataform arranges the editors and their labels corresponding to the fields in layout. However, to enable the floating label layout set `LayoutType` property of `SfDataForm` or `DataFormItem` to `TextInputLayout`.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        LayoutType="TextInputLayout">
+    </dataForm:SfDataForm>
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.LayoutType = DataFormLayoutType.TextInputLayout;
+
+{% endhighlight %}
+{% endtabs %}
+
+### Changing layout type of the DataFormItem
+You can change layout type by using the `LayoutType` property of [DataFormItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItem.html) and it will be handled in the [GenerateDataFormItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.SfDataForm.html#Syncfusion_Maui_DataForm_SfDataForm_GenerateDataFormItem) event.
```

Comment:
You can change layout type for the editor by using the

Reply:
Changed content